e2e-test/src/tests/cleanUp.test.js

```diff
@@ -0,0 +1,106 @@
+import {
+  vBtn,
+  vListTile,
+  vIcon,
+} from 'jest-puppeteer-vuetify';
+import { CLIENT_URL } from '../util';
+
+
+describe('relabel dataset', () => {
+  it('make dataset ready for analysis with 2 clicks', async () => {
+    await Promise.all([
+      page.goto(CLIENT_URL),
+      page.waitForNavigation({ waitUntil: 'networkidle0' }),
+    ]);
+    await expect(page).toClickXPath(vBtn('Your Datasets'));
+
+    const fileInputElementHandle = await page.waitForXPath("//input[@type='file']");
+    await fileInputElementHandle.uploadFile('../e2e-test/data/cleanUpTest.csv');
+    await page.waitForXPath(vListTile({ title: 'cleanUpTest.csv ', content: 'Dataset processed with 1 validation failures' }));
+
+    await expect(page).toClickXPath(vListTile({ title: 'cleanUpTest.csv ', content: 'Dataset processed with 1 validation failures' }) + vBtn('View Data'));
+
+    //  verify the dataset is failing
+    await page.waitForXPath(vListTile({ title: 'cleanUpTest.csv' }) + vIcon({ cssClass: 'mdi mdi-alert theme--light warning--text' }));
+
+    // select the third column(column 'C') and mark it as a 'Group'
+    await expect(page).toClickXPath("//div[@class='column-header-cell type-measurement'][contains(text(), 'C')]");
+    await expect(page).toClickXPath("//input[@value='group']");
+
+    //  verify the dataset is ready to process but needs to be more explict
```

Comment:
What does `but needs to be more explict` mean? I can see how this would verify that the dataset is ready to process but I don't see anything here about missing info.
Also, typo

Reply:
That was some personal notes that I forgot to take out.
It should just be: `//  verify the dataset is ready to process`